survive-the-monsters.js

```diff
@@ -80,15 +86,81 @@ function normalizeInput() {
     input[0] /= m;
     input[1] /= m;
 }
-function applyMovement(dt) {
-    const speed = movementConfig.speed * dt;
+
+function applyPlayerMovement(dt) {
+    const speed = movementConfig.playerSpeed * dt;
     playerX += input[0] * speed;
     playerY += input[1] * speed;
 }
+function applyEnemiesMovement(dt) {
+    const speed = movementConfig.enemiesSpeed * dt;
+    
+    for (let enemyId = 0; enemyId <= enemiesLargestId; enemyId++) {
+        if (!enemiesState[enemyId]) {
+            continue;
+        }
+        const manDistX = playerX - enemiesX[enemyId];
+        const manDistY = playerY - enemiesY[enemyId];
```

Comment:
**suggestion**: Consider using a more descriptive variable name than `enemyId` to improve code readability, such as `nextEnemyId`.